cmd/aznum2words-webapp/aznum2words-webapp.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"github.com/egasimov/aznum2words/cmd/aznum2words-webapp/api"
+	"github.com/egasimov/aznum2words/cmd/aznum2words-webapp/config"
+	"github.com/egasimov/aznum2words/cmd/aznum2words-webapp/handler"
+	"github.com/egasimov/aznum2words/cmd/aznum2words-webapp/router"
+	"github.com/jessevdk/go-flags"
+	"github.com/joho/godotenv"
+	"github.com/labstack/gommon/log"
+	"os"
+)
+
+func main() {
+	var err error
```

Comment:
Some recommendation regarding coding style:

1. Please don't Panic: 
https://github.com/uber-go/guide/blob/master/style.md#dont-panic

2. I guess no need to declare variable beforehand for holding errors

```
	_, err := flags.Parse(&config.Conf)
```